### ISSUE TITLE
Android - Nexus 5 @search method not working

### ISSUE THREAD

**Issue by kanapka94** [sagalbot/vue-select#570]
Hi,

I created searchable select with axios request for data. On macOSX, Windows in every browser everything is ok, but when I switched to mobile (device in title), @search event is not triggered.

Thank for your help

**Comment by tetianakh** [comment]
I'm facing the same issue on Android in Chrome browser. Is there any workaround?

**Comment by kanapka94** [comment]
Yes, but paintful :/ Just use better library: https://vue-multiselect.js.org.
I used that and everything was starting working.

**Comment by tetianakh** [comment]
Thank you @kanapka94!

**Comment by DinsmoreDesign** [comment]
Just ran into this myself. Interestingly enough, the example on the vue-select homepage works just fine, but none of the examples in the docs work.

**Comment by MatheusR42** [comment]
I had the same problem with 2.5 version, update to version 3 resolved this issue
